root/usr/lib/node/nethcti-server/plugins/com_nethcti_tcp/com_nethcti_tcp.js

```diff
@@ -1323,7 +1323,8 @@ function start() {
     // tls server
     tlsServer = tls.createServer({
         key: fs.readFileSync(tlsKey),
-        cert: fs.readFileSync(tlsCert)
+        cert: fs.readFileSync(tlsCert),
+        secureOptions: require('constants').SSL_OP_NO_TLSv1 | require('constants').SSL_OP_NO_TLSv1_1,
```

Comment:
This could be a breaking change for existing installations.
While the code looks good, we should add a new configuration to enable or disable old TLS versions.
The default should be backward compatible.